dynamodb/src/test/scala/zio/dynamodb/codec/DynamicValueGen.scala

```diff
@@ -73,50 +72,52 @@ object DynamicValueGen {
       case Schema.Enum21(case1, case2, case3, case4, case5, case6, case7, case8, case9, case10, case11, case12, case13, case14, case15, case16, case17, case18, case19, case20, case21, _)         => anyDynamicValueOfEnum(Chunk(case1, case2, case3, case4, case5, case6, case7, case8, case9, case10, case11, case12, case13, case14, case15, case16, case17, case18, case19, case20, case21))
       case Schema.Enum22(case1, case2, case3, case4, case5, case6, case7, case8, case9, case10, case11, case12, case13, case14, case15, case16, case17, case18, case19, case20, case21, case22, _) => anyDynamicValueOfEnum(Chunk(case1, case2, case3, case4, case5, case6, case7, case8, case9, case10, case11, case12, case13, case14, case15, case16, case17, case18, case19, case20, case21, case22))
       case Schema.EnumN(cases, _)                                                                                                                                                                  => anyDynamicValueOfEnum(Chunk.fromIterable(cases.toSeq))
-      case Schema.Sequence(schema, _, _, _)                                                                                                                                                        => Gen.chunkOfBounded(0, 2)(anyDynamicValueOfSchema(schema)).map(DynamicValue.Sequence(_))
+      case Schema.Sequence(schema, _, _, _, _)                                                                                                                                                     => Gen.chunkOfBounded(0, 2)(anyDynamicValueOfSchema(schema)).map(DynamicValue.Sequence(_))
       case Schema.MapSchema(ks, vs, _)                                                                                                                                                             => Gen.chunkOfBounded(0, 2)(anyDynamicValueOfSchema(ks).zip(anyDynamicValueOfSchema(vs))).map(DynamicValue.Dictionary(_))
-      case Schema.SetSchema(schema, _)                                                                                                                                                             => Gen.setOfBounded(0, 2)(anyDynamicValueOfSchema(schema)).map(DynamicValue.SetValue(_))
+      case Schema.SetSchema(ks, _)                                                                                                                                                                 => Gen.chunkOfBounded(0, 2)(anyDynamicValueOfSchema(ks)).map(DynamicValue.Sequence)
```

Comment:
Should this be `Gen.setOfBounded` so we can keep the `DynamicValue.SetValue(_)`?

Reply:
I think you are right. Updated.